### DISCUSSION TITLE
warnings.warn(f"Modifying image pixdim from {pixdim} to {norm}")

### DISCUSSION THREAD

**Discussion by MarwaAbderrahim** [Project-MONAI/MONAI#6292] [Q&A] [answered]
Hello everyone, 

I want to test Unetr 3D segmentation with the some abdominal CT dataset and i got this two errors : 

1. warnings.warn(f"Modifying image pixdim from {pixdim} to {norm}")
2.  ValueError: Sequence must have length 4, got 3. 

PS: I have not modified the code and I use the trained model with the same images.

Thank you very much for the help.

**Comment by tangy5** [comment, marked as answer]
Hi @MarwaAbderrahim , for the first, issue, that should be a warning message for pixdim spacing, it should be good. 
For the second issue, this error message typically means the spatial dimension of the data is not as expected. 
You could double check the data used is in the correct dimension (3D image and 5D tensor input). 

In addition, if you could paste complete logs of the ValueError, it will be easier to locate the issue. 

Thanks.
